scripts/xenadmin-build.ps1

```diff
@@ -29,7 +29,13 @@
 
 Param(
     [Parameter(HelpMessage = " Global build number")]
-    [int]$buildNumber
+    [int]$buildNumber,
+
+    [Parameter(HelpMessage = "Thumbrpint of the certificate to use for signing")]
+    [string]$thumbPrint,
```

Comment:
nit picky but we have suuuper small typos:


```suggestion
    [Parameter(HelpMessage = "Thumbprint of the certificate to use for signing")]
    [string]$thumbPrint,
```

Also if possible remove the leading space in the `buildNumber` param `HelpMessage`